pkg/providers/snow/creds.go

```diff
@@ -6,23 +6,29 @@ import (
 	"github.com/aws/eks-anywhere/pkg/aws"
 )
 
-type bootstrapCreds struct {
-	snowCredsB64 string
-	snowCertsB64 string
+type BootstrapCreds struct {
+	credsB64 string
+	certsB64 string
+}
+
+// Set is only used for unit test purpose.
+func (b *BootstrapCreds) Set(credsB64, certsB64 string) {
```

Comment:
It looks like the struct is exported too, did you want to make the fields exported as well. Or we can just not export the struct and only export this method if you are only using a constructor to get the secret object.